src/js/actions/libraries.js

```diff
@@ -744,11 +806,19 @@ define(function (require, exports) {
      * @return {Promise}
      */
     var afterStartup = function () {
+        CCLibraries.addLoadedCollectionsListener(handleLibrariesLoaded.bind(this));
```

Comment:
I think this is an important part of cc-libraries-api which you should explain with some comments.